code/src/components/TodoItem.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { todos } from '../reducers/todos'
+import { useSelector } from 'react-redux'
+import { Article } from '../lib/Container'
+import styled from 'styled-components'
+import moment from 'moment'
+
+export const TodoItem = ({ itemIndex }) => {
+  
+const item = useSelector(store => store.todos.list.items[itemIndex])
+const dispatch = useDispatch()
+  //Get the item from the store based on the index
+const onRemoveClicked = event => {
+  dispatch(
+    todos.actions.removeTodo({
+      itemIndex: itemIndex
```

Comment:
when key and value are the same you can write
instead of:
`itemIndex: itemIndex`
only `itemIndex`